src/features/pagination/js/pagination.js

```diff
@@ -368,8 +410,6 @@
           $scope.$on('$destroy', dataChangeDereg);
 
           var setShowing = function () {
-            $scope.showingLow = ((options.paginationCurrentPage - 1) * options.paginationPageSize) + 1;
-            $scope.showingHigh = Math.min(options.paginationCurrentPage * options.paginationPageSize, options.totalItems);
```

Comment:
The usage of `showingLow` and `showingHigh` was in the template to show the row range.
Instead of calculating the range here, I instead call the new methods from the template and add one there.